test/data/sandbox/configs/allure/allure-failed.conf.js

```diff
@@ -0,0 +1,16 @@
+exports.config = {
+  tests: './*_test_failed.js',
```

Comment:
Must be `before_suite_test_failed.js`

Reply:
resolved: https://github.com/Codeception/CodeceptJS/pull/2248/commits/b72a702897e37e935cf9cc3eed0a03e548975895